eng/scripts/preview/Download-SharedSource.ps1

```diff
@@ -65,7 +65,6 @@ $files = @(
     'OperationInternal.cs',
     'OperationInternalOfT.cs',
     'OperationPoller.cs',
-    'Optional.cs',
```

Comment:
we should also remove this file from the now-in-using-Download scripts, this script is a preview one.
And the file is not removed on your PR yet.

Reply:
It was removed in a previous pr.  Optional is not in either download script